### ISSUE TITLE
PowerNukkitX is not supported on WaterdogPE

### ISSUE THREAD

**Issue by Emotionalyear** [WaterdogPE/WaterdogPE#208]
PowerNukkitX is not supported on WaterdogPE

When I used waterdogPE to build the PowerNukkitX group server, the configuration was normal, I was able to enter the lobby server normally, and the console did not report any errors, but when I used /Server to transfer to other servers, the following prompt appeared (this should not count report an error)

![Screenshot_20220807_121053](https://user-images.githubusercontent.com/89445415/183274953-1c7aa4fb-3560-4c2f-83f4-6c34a2a51711.jpg)

I think WaterdogPE works great with PowerNukkitX, why not add PowerNukkitX as a supported server? And PowerNukkitX is a forked version based on Nukkit. This is their official website: [PowerNukkitX](https://www.powernukkitx.cn/)
Hope you can take this advice. I am from China, my English is not very good, I hope you can understand me. Thanks!

**Comment by KCodeYT** [comment]
Thats caused by the `check-login-time` configuration in the server.properties of PowerNukkitX, it checks if the client loginpacket was built at around the time the player joins. WaterdogPE builds the packet when the player connects to the Proxy (WaterdogPE) and it does not change it for every server.

TL;DR:
Just set `check-login-time` in your server.properties file to `false`.

**Comment by KCodeYT** [comment]
@Alemiz112 Issue can be closed! c:

**Comment by jameshackerbond** [comment]
can u send a download of PowernukkitX plz all I can find is just PowerNukkit

**Comment by KCodeYT** [comment]
> can u send a download of PowernukkitX plz all I can find is just PowerNukkit

https://github.com/PowerNukkitX/PowerNukkitX
https://github.com/PowerNukkitX/PowerNukkitX/releases/tag/1.19.10-r1
